docs/articles/documentation/reference/test-api/testcontroller/expect/notTypeOff.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: docs
+title: t.expect.notTypeOf Method
+permalink: /documentation/reference/test-api/testcontroller/expect/notTypeOf.html
+---
+# t.expect.notTypeOf Method
+
+Asserts that the `actual` type is not `typeName`.
+
+```text
+await t.expect( actual ).notTypeOf( typeName, message, options );
+```
+
+Parameter              | Type                                              | Description
+---------------------- | ------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------
+`actual`             | Any type | A comparison value. See [`actual` parameter value](#actual-parameter-value).
+`typeName`             | String | An unexpected type of `actual` value.
```

Comment:
> An unexpected type of `actual` value.

I used to write this way for some reason, but I guess we should come up with something better than `unexpected`.

Like

``The type that the `actual` value is not expected to have``
``The type of the `actual` value that causes the assertion to fail``

This one is good: https://github.com/DevExpress/testcafe/pull/4901/files#diff-fbd51a06a324dfaf0f49a58547ecc509R17